py3status/module.py

```diff
@@ -81,6 +82,15 @@ def load_from_namespace(module_name):
         class_inst = py_mod.Py3status()
         return class_inst
 
+    def start_module(self):
+        """
+        Start the module running.
+        """
+        # Call modules init() method if it has one
```

Comment:
This comment is out of date but could be better anyways I think, it's obvious from the code which function is being called. It should say instead why we have the hook. Just my two cents.